web/src/beta/features/PluginPlayground/Plugins/presets/layers/add-3Dtiles.ts

```diff
@@ -0,0 +1,74 @@
+import { FileType, PluginType } from "../../constants";
+
+const yamlFile: FileType = {
+  id: "layers-add-3dTiles-reearth-yml",
+  title: "reearth.yml",
+  sourceCode: `id: layers-add-3dTiles-plugin
+name: Add 3D Tiles
+version: 1.0.0
+extensions:
+  - id: layers-add-3dTiles
+    type: widget
+    name: Add 3D Tiles
+    description: Add 3D Tiles
+    widgetLayout:
+      defaultLocation:
+        zone: outer
+        section: left
+        area: top
```

Comment:
```suggestion
```
1. This widget doesn't provide a ui, so don't define this
2. In most cases, using default location is enough